src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4194,7 +4194,6 @@ private bool HasTopLevelNullabilityConversion(TypeWithAnnotations source, TypeWi
                     {
                         var lambda = (BoundLambda)operandOpt;
                         var delegateType = targetType.GetDelegateType();
-                        var unboundLambda = lambda.UnboundLambda;
                         var variableState = GetVariableState(stateForLambda);
                         Analyze(compilation,
```

Comment:
>Analyze(compilation, [](start = 24, length = 20)

Could this also be just a VisitLambda call?

Reply:
Sure, although `VisitLambda` will need an optional `initialState` parameter.

---
In reply to: [281745456](https://github.com/dotnet/roslyn/pull/35547#discussion_r281745456) [](ancestors = 281745456)